composer.lock

```diff
@@ -4025,5 +3796,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.0.0"
+    "plugin-api-version": "1.1.0"
```

Comment:
@tsteur @diosmosis @Findus23 At some point we maybe should decide to use the same composer version 🙈  We are constantly changing this, as well as adding/removing the support and funding sections.

Reply:
Indeed I just pushed the lock file for the tests as it will probably be out of date once this can be merged.

Composer 2 seems to be quite a major change so maybe we should look into it once the Matomo 4 updates have calmed down.